Cargo.toml

```diff
@@ -1,11 +1,19 @@
+[workspace]
+members = [".", "redismodule-rs-macros", "redismodule-rs-macros-internals"]
```

Comment:
is it normal to have this root member like this?

Reply:
I generally prefer to have all root members in subfolders, since it makes `cargo build` in the root of the repository equivalent to `cargo build --workspace`. 
But I know that some projects prefer to have the "key" package at the root level, so no strong preferences here.